adobe/spectrum-web-components
# fix(help-text): apply aria-live to ensure full help text is read to user

## Description
Use `aria-live="assertive"` to ensure the Help Text content (when changing) has higher screen reader priority than Textfield input.

## Related issue(s)
- fixes #3377


## How has this been tested?
    1. Go [here](https://assertive-help-text--spectrum-web-components.netlify.app/components/textfield/#help-text)
    2. Turn on your screen reader
    3. Focus the `Stay "Positive"` text field
    4. Type or delete anything and then keep typing
    5. Ensure that the screen reader reads ALL of the updated help text when moving on and off of the value `Positive`

## Types of changes

## Checklist
